src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -850,15 +856,16 @@
     <value>Made with 💗 by Microsoft and the PowerToys community.</value>
   </data>
   <data name="ColorPickerOnly.Text" xml:space="preserve">
-    <value>Color picker only</value>
+    <value>Color Picker only</value>
+    <comment>do not loc the product name</comment>
   </data>
   <data name="ColorPickerOnly_Description.Text" xml:space="preserve">
-    <value>Pick a color and copy the preferred format to clipboard</value>
+    <value>Pick a color from the screen and copy formatted value to clipboard</value>
   </data>
   <data name="EditorFirst_Description.Text" xml:space="preserve">
-    <value>Open the editor to view previously picked color and to pick new colors</value>
+    <value>Open directly into the editor mode</value>
   </data>
   <data name="ColorPicker_ColorFormatsDescription.Text" xml:space="preserve">
-    <value>Individual color formats can be shown or hidden in the list below. Change the order of appearance by dragging.</value>
+    <value>Additional color formats (Change the order by dragging)</value>

```

Comment:
How should the user who isn't involved in the dev process know, that this is for the editor?!


```suggestion
    <value>Color formats in Editor (Change the order by dragging)</value>
```

Reply:
@niels9001, @martinchrzan
Can we commit this suggested change?

Reply:
this should be merged to the wpf branch then to the main branch so we don't loc twice

Reply:
i unified language here